lib/takeaway.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'menu'
+require_relative 'customer_order'
+require_relative 'twilio'
+
+class Takeaway
+
+  attr_reader :menu, :cust_order, :sms_handler
+
+  def initialize(menu_klass: Menu.new, sms_klass: TwilioSMS.new)
```

Comment:
this is failing your CI - just pass in a double instead of the TwilioSMS instance.

only use _klass if you are actually passing a class - you are passing instances here, not classes.